yarn.lock

```diff
@@ -5803,13 +5205,6 @@ npm-run-path@^4.0.0, npm-run-path@^4.0.1:
   dependencies:
     path-key "^3.0.0"
 
-nth-check@^1.0.2:
```

Comment:
Fixes dependabot issue [Inefficient Regular Expression Complexity in nth-check #3](https://github.com/SalesforceFoundation/OutboundFundsModule/security/dependabot/3) as nth-check is no longer used and removed from `yarn.lock`.  Do a find for `nth-check@` to verify it's not used.